server/src/main/java/org/elasticsearch/cluster/metadata/ComposableIndexTemplate.java

```diff
@@ -132,6 +138,11 @@ public ComposableIndexTemplate(StreamInput in) throws IOException {
         } else {
             this.dataStreamTemplate = null;
         }
+        if (in.getVersion().after(Version.V_7_10_0)) {
```

Comment:
Typically, we set this to V.8_0_0 until after we backport

Reply:
Ah, I wasn't sure what the convention was here. Does that mean that the backport has `V_7_10_0` and master has `V_8_0_0`, or do we also adjust master?